frontend-react/src/components/shared/Table/Table.module.scss

```diff
@@ -0,0 +1,53 @@
+@use 'uswds-core' as *;
```

Comment:
I think the directory structure should be reversed here: `frontend-react/src/shared/components`.  This'll allow for other `shared` directories like `frontend-react/src/shared/utils`, `frontend-react/src/shared/hooks`, et cetera.

Reply:
Hmmm I feel like that's a little redundant (having two have two `/components` folders) and a little confusing.  I could see shared utility functions, hooks etc being in a root folder `/shared` folder, but these are components, and I think they should all live together in a singular `/components` folder.